integrationlink/create.go

```diff
@@ -0,0 +1,54 @@
+package integrationlink
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+
+	"gopkg.in/errgo.v1"
+
+	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/cli/io"
+	"github.com/Scalingo/go-scalingo"
+)
+
+func Create(app string, integrationType scalingo.SCMType, integrationURL string, params scalingo.SCMRepoLinkParams) error {
+	u, err := url.Parse(integrationURL)
+	if err != nil || u.Scheme == "" || u.Host == "" || u.Path == "" {
+		return errors.New("source repository URL is not valid")
+	}
+
+	c, err := config.ScalingoClient()
+	if err != nil {
+		return errgo.Notef(err, "fail to get Scalingo client")
+	}
+
+	integration, err := c.SCMIntegrationsShow(string(integrationType))
+	if err != nil {
+		return errgo.Notef(err, "fail to get the integration")
+	}
+
+	repoLink, err := c.SCMRepoLinkShow(app)
```

Comment:
Wont SCMRepoLinkShow send an error if not SCM is configured (like a 404 ?)

Reply:
Surprisingly, no..